src/libs/Visibility/index.native.js

```diff
@@ -8,6 +8,13 @@ import {AppState} from 'react-native';
  */
 const isVisible = () => AppState.currentState === 'active';
 
+/**
+ * @returns {Boolean}
+ */
+function hasFocus() {
+    return true;
+}
```

Comment:
Assuming it has focus is weird?

Reply:
I think it's fine since checking isVisible should be enough on the native case.

Reply:
I see your point @Santhosh-Sellavel. In the future, we might want this function to behave as `onResume`-`onPause` lifecycle on Android. I mean, for instance, sometimes the App might be foregrounded and partially visible but not focused because there is a system alert on top of it. But I also do agree with @luacmartins that this might be out of scope for now.